rasterio/warp.py

```diff
@@ -374,6 +374,9 @@ def calculate_default_transform(
     resolution: tuple (x resolution, y resolution) or float, optional
         Target resolution, in units of target coordinate reference
         system.
+    dimensions: tuple (width, height), optional
+        Output file size in pixels and lines. Cannot be used together
+        with resolution.
```

Comment:
We're already passing width and height, how is this parameter different?

Reply:
It makes sense. Could you propose possible name of this option?